kalite/i18n/base.py

```diff
@@ -0,0 +1,361 @@
+import errno
```

Comment:
The only thing that was changed was unused functions were deleted, and things that relied on old content backend stuff was modified - don't think any new code is in this file though.